polaris.shopify.com/content/design/colors.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Color
+description: Color is a powerful tool that helps merchants quickly navigate and manage their businesses in the Shopify Admin.
+icon: ColorsMajor
+keywords:
+  - visual patterns
+  - color strategy
+  - color use
+---
+
+![A rainbow of colors created by small bricks](/images/design/colors/color-intro@2x.png)
+
+---
+
+# {frontmatter.title}
+
+<Lede>{frontmatter.description}</Lede>
+
+<DirectiveCard status="Caution">
```

Comment:
we have a [status banner component](https://github.com/Shopify/polaris/blob/a15de9733e4ffe01c336fc4184308c45db30968e/polaris.shopify.com/src/components/StatusBanner/StatusBanner.tsx#L10) that is probably better for this callout on the state of these pages

Reply:
I think the way the `Caution` directive grabs attention is appropriate. I'll leave it as-is for this, it's just going to be there for a few days.